### PR TITLE
bpo-37578: glob.glob -- added include_hidden parameter

### PR DESCRIPTION
**Pull request by akulakov**
https://bugs.python.org/issue37578

Automerge-Triggered-By: GH:asvetlov

### CODE REVIEW

**Comment by akulakov** [comment]
I will add a news entry later today

**Comment by asvetlov** [comment]
Looks good so far.

File: Doc/library/glob.rst [hunk 1]
```diff
@@ -36,7 +36,8 @@ For example, ``'[?]'`` matches the character ``'?'``.
    The :mod:`pathlib` module offers high-level path objects.
 
 
-.. function:: glob(pathname, *, root_dir=None, dir_fd=None, recursive=False)
+.. function:: glob(pathname, *, root_dir=None, dir_fd=None, recursive=False,
+   include_hidden=False)
```
Comment (asvetlov) [inline Doc/library/glob.rst]:
```suggestion
.. function:: glob(pathname, *, root_dir=None, dir_fd=None, recursive=False, \
                   include_hidden=False)
```

File: Doc/library/glob.rst [hunk 2]
```diff
@@ -77,8 +80,12 @@ For example, ``'[?]'`` matches the character ``'?'``.
    .. versionchanged:: 3.10
       Added the *root_dir* and *dir_fd* parameters.
 
+   .. versionchanged:: 3.11
+      Added the *include_hidden* parameter.
+
 
-.. function:: iglob(pathname, *, root_dir=None, dir_fd=None, recursive=False)
+.. function:: iglob(pathname, *, root_dir=None, dir_fd=None, recursive=False,
+   include_hidden=False)
```
Comment (asvetlov) [inline Doc/library/glob.rst]:
```suggestion
.. function:: iglob(pathname, *, root_dir=None, dir_fd=None, recursive=False, \
                    include_hidden=False)
```

**Review by asvetlov — APPROVED** [review]

**Comment by akulakov** [comment]
@asvetlov thanks for reviewing!

**Comment by asvetlov** [comment]
@akulakov you are welcome!

**Pull request merged**